eng/tools/ci/uv/whl.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env -S uv run --script
+# /// script
+# requires-python = ">=3.9"
+# dependencies = [
+#   "wheel==0.45.1",
+#   "packaging==24.2",
+#   "urllib3==2.2.3",
+#   "tomli==2.2.1",
+#   "build==1.2.2.post1",
+#   "pytest==8.3.5",
+#   "pytest-cov==5.0.0",
+#   "azure-sdk-tools",
+#   "setuptools",
+#   "pytest-asyncio==0.24.0",
+#   "pytest-custom-exit-code==0.3.0",
+#   "pytest-xdist==3.2.1",
+#   "coverage==7.6.1",
+#   "bandit==1.6.2",
+#   "pyproject-api==1.8.0",
+#   "jinja2==3.1.6",
+#   "json-delta==2.0.2",
+#   "readme-renderer==43.0",
+#   "python-dotenv==1.0.1",
+#   "pyyaml==6.0.2",
+#   "six==1.17.0",
+# ]
+#
+# [tool.uv.sources]
+# azure-sdk-tools = { path = "../../../../tools/azure-sdk-tools", editable = true }
```

Comment:
I should back this out and leave it to be updated manually when debugging a uv check